app/main.py

```diff
@@ -1,6 +1,18 @@
-from typing import Callable
+from functools import wraps
+from typing import Callable, Any
 
 
-def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+def cache(func: Callable) -> Callable[..., Any]:
```

Comment:
There is no need to specify that the wrapper could return any argument, this annotation is enough  
```suggestion
def cache(func: Callable) -> Callable:
```